app/views/pieces/show.html.erb

```diff
@@ -0,0 +1,87 @@
+<p id="notice"><%= notice %></p>
+
+<div class="chessboard">
+  <div id="row_7">
+    <div class="white"><%= render_piece(0, 0) %></div>
+    <div class="black"><%= render_piece(0, 1) %></div>
+    <div class="white"><%= render_piece(0, 2) %></div>
+    <div class="black"><%= render_piece(0, 3) %></div>
+    <div class="white"><%= render_piece(0, 4) %></div>
+    <div class="black"><%= render_piece(0, 5) %></div>
+    <div class="white"><%= render_piece(0, 6) %></div>
+    <div class="black"><%= render_piece(0, 7) %></div>
+  </div>
```

Comment:
@mscoutermarsh I created the instance variable `@current_piece` like we discussed in our team meeting the other day, but now I'm unsure of how to actually use it in the view. I know I need to say "if `Piece` == `@current_piece` then highlight that piece's square", but I'm not sure how, and I'm not asking "Jeeves"(google) the right questions to find an answer.

Reply:
`@current_piece` will also be accessible from within the helper `render_piece`. Does that make things easier 😄 ?

Reply:
Ahhh, that should make things easier. Certainly makes sense. I'll let you know how it goes though!